### DISCUSSION TITLE
websocket 需要修改源码吗

### DISCUSSION THREAD

**Discussion by Wang-Junwen** [kbengine/kbengine#1403] [Q&A]
做小游戏，是websocket协议，kbe是需要修改配置 还是要改源码，还是只配置nginx反向代理

客户端是unity ，添加了websocket的连接，但是服务端收到PacketReader::processMessages: not found msgID=0, msglen=1022, from

**Comment by huhubis** [comment]
只配置nginx反向代理

**Reply by Wang-Junwen** [reply]
客户端需要修改什么吗 我现在是onhello成功了 但是发送的logindata 服务端没有回消息 ，切回tcp就没问题 用ws就这样

**Reply by huhubis** [reply]
参考这个 https://github.com/jay602/stone_war

**Comment by qhf566** [comment]
想找一款成熟的手游产品源码过来我们深度二次开发(全新换套美术，代码也都从新货混淆及优化整一遍)。意向最好是重度的mmo、arpg、角色扮演或动作手游，其他类型的游戏也可以，必须是高品质大厂近5年内做的产品，国内厂商或国外厂商开发的都可以，产品不能在国内运营过的。可以在国外运营过或运行中。准备了足够夸张的米给到你。如朋友那边有可以给推荐一下有米，不墨迹,鹅浩809996677。
